react-app/src/pages/TablePage.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react"
+
+export default function TablePage() {
+    const [TableContent, setContent] = useState<[number, string, number][]>([[0, "Marcel", 130]])
```

Comment:
You could use a type definition from TypeScript, I also suggest to make it object not a list for better readability

Reply:
Good point about making it an object. Will address it tomorrow.